iac/terraform/aws/modules.tf

```diff
@@ -56,7 +56,7 @@ module "donor_search_router" {
 }
 
 module "donor_search_sf" {
-  source                       = "./donor_search_sf"
-  environment                  = var.environment
-  dynamodb_table_arn           = module.database.dynamodb_table_arn
+  source             = "./donor_search_sf"
```

Comment:
Is it appropriate to merge donor_search_router and donor_search_sf module and create a module named donor_search?

Reply:
we can merge these two modules. But what i thought was, we are gonna add more things in the `donor_search_sf`, we have two more ticket on that. so it might get crowded in the future

Reply:
Updated in 295421db8a08f3ed9c398dc5222becf78f98d123